.github/workflows/node.js.yml

```diff
@@ -76,6 +76,17 @@ jobs:
         ssh-add - <<< '${{ secrets.PRIVATE_SSH_KEY }}'
         git clone git@github.com:MTES-MCT/ecobalyse-private.git
 
+    - name: Checkout specific ecobalyse-private branch
+      if: ${{ github.event_name == 'pull_request' }}
+      env:
+        LAST_COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
+      run: |
+        eval `ssh-agent -s`
+        # Private ssh key used by this Github action to clone the private repo
+        # The corresponding public key must be set as a deploy key of the private repo
+        ssh-add - <<< '${{ secrets.PRIVATE_SSH_KEY }}'
+        ./bin/checkout-ecobalyse-private-branch.sh $LAST_COMMIT_SHA
```

Comment:
I understand that this is done during the build?
I created a PR with a merge of this one and the organic_tomato one (#665) to test the behaviour. I initially forgot to specify the `ecobalyse_data: <commit>` in the PR, and corrected while it was running. The test is failing by not finding a new process id which should be in the private repo. So we might need a way to detect and restart the tests/checks after modifying the description of the PR, a bit like the semantic PR title check. But would it be relevant?
```
on:
  pull_request_target:
    types:
      - opened
      - edited
      - synchronize
```

Reply:
Now it's computed on every test run. It was a bug in branch name detection, it should be fixed now.